.github/workflows/convential-commits-action.yml

```diff
@@ -0,0 +1,12 @@
+name: Conventional Commits
+
+on: push
+
+jobs:
+  build:
+    name: Conventional Commits
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
This line triggers a warning in the check, action should be updated to the latest version.